### PR TITLE
Changes related to clowder db integration.

### PR DESCRIPTION
**Pull request by msvinaykumar**

### CODE REVIEW

File: src/main/resources/config.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
```
Comment (dinogun) [inline src/main/resources/config.json:1]: Do you expect this file to have non database related config as well? Just thinking from the file naming perspective

File: src/main/resources/config.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    "adminPassword": "admin",
+    "adminUsername": "admin",
+    "hostname": "postgres-service",
+    "name": "kruizeDB",
+    "password": "admin",
+    "port": 5432,
+    "sslMode": "require",
+    "username": "admin"
+  }
+}
```
Comment (dinogun) [inline src/main/resources/config.json:12]: Miising EoF

File: src/main/java/com/autotune/dbactivites/init/KruizeHibernateUtil.java [hunk 3]
```diff
@@ -28,15 +37,65 @@ public class KruizeHibernateUtil {
...
+                        ":" + databaseObj.getInt(AnalyzerConstants.DBConstants.PORT) +
+                        "/" + databaseObj.getString(AnalyzerConstants.DBConstants.NAME);
+
+                configuration.setProperty("hibernate.connection.url", connectionURL);
+                configuration.setProperty("hibernate.connection.username", databaseObj.getString(AnalyzerConstants.DBConstants.USERNAME));
+                configuration.setProperty("hibernate.connection.password", databaseObj.getString(AnalyzerConstants.DBConstants.PASSWORD));
+                sfTemp = configuration.buildSessionFactory();
+                LOGGER.debug("DB build session is successful !");
+            }else{
+                LOGGER.debug("DB build session failed !");
```
Comment (dinogun) [inline src/main/java/com/autotune/dbactivites/init/KruizeHibernateUtil.java:97]: Should these be `LOGGER.info`?

**Comment by dinogun** [comment]
@msvinaykumar also I think the requirement was that this needs to be over a volume mount rather than packaged inside the docker image

**Comment by msvinaykumar** [comment]
> @msvinaykumar also I think the requirement was that this needs to be over a volume mount rather than packaged inside the docker image

@dinogun please check now I removed config.json pacaking under docker image. now everything is one single yaml file with support of volumemount and it supports community too.

**Review by dinogun — APPROVED** [review]
LGTM

File: manifests/crc/BYODB-installation/minikube/kruize-crc-minikube.yaml [hunk 4]
```diff
@@ -25,6 +45,9 @@ spec:
         - name: kruize
           image: kruize/autotune_operator:0.0.9_rm
```
Comment (chandrams) [inline manifests/crc/BYODB-installation/minikube/kruize-crc-minikube.yaml:46]: Should we update the image tag to 0.0.10_rm?

File: manifests/crc/BYODB-installation/openshift/kruize-crc-openshift.yaml [hunk 5]
```diff
@@ -25,6 +45,9 @@ spec:
         - name: kruize
           image: kruize/autotune_operator:0.0.9_rm
```
Comment (chandrams) [inline manifests/crc/BYODB-installation/openshift/kruize-crc-openshift.yaml:46]: Should we update the image tag to 0.0.10_rm?

**Review by chandrams — APPROVED** [review]
LGTM

File: manifests/crc/BYODB-installation/minikube/kruize-crc-minikube.yaml (hunk 4 above)
Comment (dinogun) [reply manifests/crc/BYODB-installation/minikube/kruize-crc-minikube.yaml]: @chandrams as per the process we have to run tests on 0.0.10_mvp and if the tests are successful, tag and push 0.0.10_rm and then change this to 0.0.10_rm. @msvinaykumar I'd suggest you revert it back. Sorry about that

Comment (chandrams) [reply manifests/crc/BYODB-installation/minikube/kruize-crc-minikube.yaml]: Thanks Dino for pointing this out. @msvinaykumar - Please revert it back

**Pull request merged**